frontend/src/scripts/font_awesome_icons.ts

```diff
@@ -0,0 +1,7 @@
+import { library } from '@fortawesome/fontawesome-svg-core';
+import { faChevronRight, faCheck } from '@fortawesome/free-solid-svg-icons';
+
+library.add({
```

Comment:
Awesome. Maybe we should just go with that. I waas questioning the library `fortawesome` because I thought it might be heavy. But seems like something you the developper can manage.